src/hardware/mame/ymf262.cpp

```diff
@@ -2353,12 +2353,20 @@ static void OPL3ResetChip(OPL3 *chip)
 /* 'rate'  is sampling rate  */
 static OPL3 *OPL3Create(device_t *device, int clock, int rate, int type)
 {
+	// Guard
+	if (device == nullptr) {
+		return 0;
```

Comment:
`return nullptr`?

Reply:
Got it. Also added a couple other `nullptr` returns instead of `0` too.